frontend/src/components/single-article/style.css

```diff
@@ -1216,7 +1212,7 @@ DESKTOP CSS SETTTINGS. At this point, able to reuse some of the sections and inc
     height: 16px;
   }
 
-  .article-content .date-and-author {
+  .article-content {
```

Comment:
Same thing goes for this but this is for the DESKTOP version: This may affect the whole article-content styling (already defined earlier on Line 761). So may good idea to delete this altogether since this was originally supposed to format specifically for the date and author section which we have since removed.